### ISSUE TITLE
[stable/cluster-autoscaler] some params provided through extraArgs doesn't apply

### ISSUE THREAD

**Issue by 0lmi** [helm/charts#14972]
**Describe the bug**
When you specify `skip-nodes-with-local-storage` and/or `skip-nodes-with-system-pods` through `extraArgs` then empty values are applied.

**Version of Helm and Kubernetes**:
K8s: v1.12.6-eks-d69f1b
Helm: v2.14.1

**Which chart**:
v.0.13.3

**What happened**:
When you install cluster-autoscaler with providing `--set extraArgs.skip-nodes-with-local-storage=false` and/or `--set extraArgs.skip-nodes-with-system-pods=false` then `cluster-autoscaler-aws-cluster-autoscaler` deployment is created with `--skip-nodes-with-local-storage` and/or `--skip-nodes-with-system-pods` parameters but without `=$value`. This looks like the following:
```
$ kubectl -n kube-system describe deployments. cluster-autoscaler-aws-cluster-autoscaler | grep -A8 "Command:"
    Command:
      ./cluster-autoscaler
      --cloud-provider=aws
      --namespace=kube-system
      --node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled,kubernetes.io/cluster/olmi-migration-test
      --logtostderr=true
      --skip-nodes-with-local-storage
      --skip-nodes-with-system-pods
      --stderrthreshold=info
```

**What you expected to happen**:
Deployment should be created with both `key` and `value` instead of just `key`.

**How to reproduce it** (as minimally and precisely as possible):
Install cluster autoscaler like the following on AWS EKS:
```
helm install stable/cluster-autoscaler \
  --name cluster-autoscaler \
  --namespace kube-system \
  --set awsRegion=eu-central-1 \
  --set sslCertPath=/etc/kubernetes/pki/ca.crt \
  --set rbac.create=true \
  --set autoDiscovery.clusterName=olmi-migration-test \
  --set extraArgs.skip-nodes-with-local-storage=false \
  --set extraArgs.skip-nodes-with-system-pods=false
```
and check the deployment:
```
kubectl -n kube-system describe deployments. cluster-autoscaler-aws-cluster-autoscaler | grep -A8 "Command:"
    Command:
      ./cluster-autoscaler
      --cloud-provider=aws
      --namespace=kube-system
      --node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled,kubernetes.io/cluster/olmi-migration-test
      --logtostderr=true
      --skip-nodes-with-local-storage
      --skip-nodes-with-system-pods
      --stderrthreshold=info
```

**Anything else we need to know**:

**Comment by yurrriq** [comment]
In this case you need to specify boolean values as strings, e.g.

```yaml
extraArgs:
  skip-nodes-with-local-storage: 'false'
```

**Comment by 0lmi** [comment]
It's fixed after #12998 is merged (version 0.14.2).
